orderer/consensus/smartbft/util_network_test.go

```diff
@@ -92,6 +97,7 @@ func (ns *NetworkSetupInfo) SendTxToAllAvailableNodes(tx *cb.Envelope) error {
 			ns.t.Logf("Error occurred during sending tx to node %v: %v", idx, err)
 		}
 		errorsArr = append(errorsArr, err)
+		ns.t.Logf("Tx to node %v was sent successfully", idx)
```

Comment:
there might have been an error here. 
you also append nil errors.
why not do - if there is an error, append it and continue?